### PR TITLE
Docs/contributing to discussion/25

### PR DESCRIPTION
**Pull request by capsulecorplab**
This pull request (PR) builds on top of #28, with the addition of the [Contributing to discussions](https://github.com/ODU-CGA-CubeSat/flight-software/commit/d5e0b1418bc8c823997141d7e408abfcf6cfe2dd) page

### CODE REVIEW

File: docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
+
+---
+title: "Contributing to discussions"
```
Comment (kchiu002) [inline docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc:3]: Title appears to be in the wrong format.

File: docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+weight: 1
+description: >
+  How to contribute to discussions & issues
+---
+
+The https://github.com/odu-cga-cubesat/flight-software/discussions[Discussions] and https://github.com/odu-cga-cubesat/flight-software/issues[Issues] sections help fascilitate invaluable feedback and discourse in the development and maintainability of the ODU/CGA CubeSat flight software.
```
Comment (kchiu002) [inline docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc:9]: "fascilitate" misspelled.

File: docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+. Specify a title for the discussion topic, e.g., _Design constraints of mission concept_.
+. Write a brief description of the scope of the discussion, e.g., _This discussion is to capture design constraints of the mission concept_.
+. Click *Start discussion*.
+
+See https://docs.github.com/en/discussions/quickstart#creating-a-new-discussion[Creating a new discussion].
+
+== Opening an issue
+
+https://github.com/odu-cga-cubesat/flight-software/issues[Issues] specify tasks that allow maintainers to capture & resolve missing or incorrect content, as well as implement new features or bug fixes.
+Some issues are grouped by https://github.com/odu-cga-cubesat/flight-software/milestones[Milestones] - a collection of issues that relate to a specific deliverable, s.a., a https://github.com/odu-cga-cubesat/flight-software/releases[Release].
```
Comment (kchiu002) [inline docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc:29]: The "s.a.," supposed to be a "such as"?

File: docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+Issues are tagged https://github.com/odu-cga-cubesat/flight-software/labels[Labels] in order to identify their associated domain.
```
Comment (kchiu002) [inline docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc:30]: Change "are tagged" to "are tagged with".

**Review by kchiu002 — COMMENTED** [review]
Review complete.  First file comments are associated with Pull Request #28.  See #28 for comments on first file.

File: docs/content/contribute/how-to-contribute/_index.adoc [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+== Contribute as a technical writer
+
+A technical writer is a person who maintains the state of the documentation. Technical writers work with users, developers, and/or subject-matter experts in order to document the intended design and behavior of the codebase, to maintain a consistent format, style, and tone, as well as to verify the contents readablility and usability.
+
+Technical writers can primarily contribute by link:./contributing-to-documentation[Contributing to documentation].
+
+== Contribute as a developer
+
+A developer is a person who impements the codebase, as specified by the documentation. Developers should refer to the documentation to implement and test the intended design and behavior of the software in development.
```
Comment (GonzaBasket98) [inline docs/content/contribute/how-to-contribute/_index.adoc:26]: Typo: "impements" should be "implements"

**Review by GonzaBasket98 — COMMENTED** [review]
Quick typo

**Comment by capsulecorplab** [comment]
converting to draft 'til we can close #43

File: docs/content/contribute/how-to-contribute/_index.adoc (hunk 5 above)
Comment (capsulecorplab) [reply docs/content/contribute/how-to-contribute/_index.adoc:26]: resolving in https://github.com/ODU-CGA-CubeSat/mission-conops/pull/28/commits/a8fd7480401dbe323c483ba99755407f58910fcd

File: docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc (hunk 2 above)
Comment (capsulecorplab) [reply docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc:9]: resolving in https://github.com/ODU-CGA-CubeSat/mission-conops/pull/28/commits/5a88a8cc9653ad850cc0ae965e81305b807d2e92

File: docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc (hunk 3 above)
Comment (capsulecorplab) [reply docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc:29]: resolving in https://github.com/ODU-CGA-CubeSat/mission-conops/pull/28/commits/14dadf6c6cf2017b7c1d162d1c57e18419235463

File: docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc (hunk 4 above)
Comment (capsulecorplab) [reply docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc:30]: resolving in https://github.com/ODU-CGA-CubeSat/mission-conops/pull/28/commits/14dadf6c6cf2017b7c1d162d1c57e18419235463

**Comment by capsulecorplab** [comment]
closing this PR, since its associated issue is being resolved in https://github.com/ODU-CGA-CubeSat/mission-conops/pull/28

**Pull request closed without merging**